Cloud-Engineering/L4 - Lead Cloud Engineer.md

```diff
@@ -0,0 +1,75 @@
+- I make informed architectural decisions considering cross-cutting concerns such as infrastructure, security, scalability, and maintainability.
+- I raise potential problems with my team before getting blocked, ask for help when needed, and offer help when I can assist.
+- I integrate reliability engineering principles into my work, focusing on automation, observability, and efficient incident response.
+- I implement and advocate for cloud security best practices, ensuring the protection of our infrastructure and data.
+
+**Examples:**
+
+- I led a medium-sized project for Octopus Cloud and product teams from pitch to production, collaborating with my team and stakeholders.
+- I made pragmatic decisions to ship a product, engaging stakeholders and keeping them informed.
+- People seek my opinion for technical decisions due to my proven track record of wise choices.
+- I guided my team’s decision-making around operational and security concerns, balancing delivery and quality.
+
+## 🌱 Teaching and Mentoring
+
+- I consistently help new hires and team members to "level up" and become more proficient over time.
+- I fuel my team's desire to learn and provide mentorship on effective use of containers, virtualization, and cloud security practices.
+- I define, document, and evolve efficient practices for my team, leading to high-quality outcomes.
+- I challenge and coach my peers, helping them grow and focus on valuable contributions.
+
+**Examples:**
+
+- I provided training and mentoring for team members, helping them round out their skill sets.
+- I shared my on-the-job learning and experiences to enhance the effectiveness of others in their roles.
+- I broke up a project to align appropriate challenges for each teammate, aiding their growth.
+- I had difficult conversations with teammates, challenging them directly while showing personal care.
+
+## 🧭 Culture and Leadership
+
+- I collaborate on technical direction within already-defined projects and maintain/improve quality across the stack.
+- I take on necessary and unglamorous work to get things done and take responsibility for my mistakes while learning from others.
+- I manage difficult conversations and tactfully challenge others, improving team productivity and delivery.
+- I coordinate projects that require input from multiple teams and diverse technical roles, ensuring successful outcomes.
+- I advocate for and implement best practices in monitoring, alerting, and incident management, enhancing the reliability and performance of our systems.
+- I ensure compliance with security policies and manage risk assessments, incident responses, and security audits.
+
+**Examples:**
+
+- I took on a significant share of unplanned work and other "housekeeping" tasks.
+- I spotted a contentious issue early, facilitated a resolution, and ensured positive outcomes.
```

Comment:
```suggestion
- I spotted a contentious issue early, facilitated a resolution, and ensured positive outcomes.
- I identified waste in our team processes, formulated a new approach, and got the rest of the team to buy into a new and better way of doing things.
```

💡 I suggest copying this example from "Lead Software Engineer", which mentions taking the initiative to reduce the ongoing cost of "toil" for the team. I think this could be particularly relevant for Lead Cloud Engineers who are in teams with "operations" responsibilities.